### ISSUE TITLE
Compare the metadata fields for kidney datasets to the fields the HCA metadata schema

### ISSUE THREAD

**Issue by zperova** [HumanCellAtlas/metadata-schema#1210]
**Description**

 

As an HCA wrangler who works on archiving HCA data, I would like to know how the fields [GEO and AE collected](https://drive.google.com/drive/folders/11-4mbMmZD_WOMzewgFdbGOUClQux6YjW) for the identified [kidney datasets](https://docs.google.com/spreadsheets/d/1gY--ckT04Jc3xJ9atvNbm0KVty86CwKtR3cz_aau0L4/edit#gid=2049210213) compares to the HCA metadata schema.  

**Acceptance Criteria**

- [x] Extract AE and GEO metadata from native format to usable format
- [x] Perform a mapping of the fields collected by GEO and AE for the selected Kidney datasets and deposit in the [google drive folder](https://drive.google.com/drive/folders/11-4mbMmZD_WOMzewgFdbGOUClQux6YjW) 
- [ ] Assess any other useful metadata from the publications

**Comment by zperova** [comment]
I made a column in the spreadsheet for wranglers to put their name when they start working on a certain dataset: https://docs.google.com/spreadsheets/d/1gY--ckT04Jc3xJ9atvNbm0KVty86CwKtR3cz_aau0L4/edit#gid=2049210213

**Comment by ami-day** [comment]
Thanks that's helpful. I plan on working on a script to list the dataset ids and their unique fields in a table - it might be good to see the commonalities in fields as an alternative to going through each manually (and probably useful for future added GEO/AE datasets)

**Comment by ami-day** [comment]
Hi, I completed the metadata fields for GEO datasets GSE114156 and GSE109564 which are both associated with the following publication by Humphreys et al.:
"Single-Cell Transcriptomics of a Human Kidney Allograft Biopsy Specimen Defines a Diverse Inflammatory Response".

It would be great if this could be reviewed? @mshadbolt @zperova @ESapenaVentura. Here is the file location: https://drive.google.com/drive/folders/118kh4wiHmn4Oz9n1-WZueaxm-8XuCMkA. 

There was already a filled-in sheet for GSE109564 in finished projects, so I copied that info. over into the combined sheet.
